### PR TITLE
[TAS-456] 🚸 Support two different minting methods

### PR DESCRIPTION
**Pull request by AuroraHuang22**
網頁擷取_6-10-2023_11358_localhost
網頁擷取_6-10-2023_1148_localhost
網頁擷取_6-10-2023_11446_localhost
網頁擷取_6-10-2023_11519_localhost

### CODE REVIEW

File: pages/mint-nft/index.vue [hunk 1]
```diff
@@ -196,6 +278,10 @@ const iscnIdInput = ref(route.query.class_id || route.query.iscn_id || '')
 const iscnOwner = ref('')
 const iscnCreateData = ref<any>(null)
 const iscnData = ref<any>(null)
+const nftIdPrefix = ref('BOOKSN')
```
Comment (williamchong) [inline pages/mint-nft/index.vue:281]: what does `BOOKSN` mean here?

File: pages/mint-nft/index.vue [hunk 2]
```diff
@@ -303,6 +391,73 @@ function onISCNFileChange (event: Event) {
...
 
+async function onClickMintByInputting () {
+  isLoading.value = true
+  step.value = 3
+  const { contentMetadata } = iscnData.value
+
+  const nftClassData = {
+    name: contentMetadata.name,
+    description: contentMetadata.description,
+    symbol: 'CNR',
```
Comment (williamchong) [inline pages/mint-nft/index.vue:402]: What does `CNR` mean here?

File: pages/mint-nft/index.vue [hunk 3]
```diff
@@ -68,31 +73,125 @@
...
+        <div class="container">
+          <h3>Mint NFT by filling required information</h3>
+          <div style="display: flex; flex-direction: column; gap: 8px">
+            <div>
+              <label>NFT ID Prefix: </label>
+              <input v-model="nftIdPrefix" placeholder="English only ex.MoneyVerse">
+            </div>
+            <div>
+              <label>Number of NFT to mint: </label>
+              <input v-model="nftMintCount" placeholder="0-100">
```
Comment (williamchong) [inline pages/mint-nft/index.vue:158]: Can use `type=number` and set min max to improve ux

File: pages/mint-nft/index.vue (hunk 1 above)
Comment (edmondyu) [reply pages/mint-nft/index.vue:281]: Book Serial Number.  This label is not important and should not hold the user back for even 1 second, so give it a default.

File: pages/mint-nft/index.vue (hunk 2 above)
Comment (edmondyu) [reply pages/mint-nft/index.vue:402]: I don't know... should be from one of the early sample or template

**Review by williamchong — APPROVED** [review]

**Pull request merged**